test/unit/testExecutorRoles.js

```diff
@@ -63,6 +63,132 @@ describe('ExecutorRoles', () => {
         });
     });
 
+    describe('isComplete()', () => {
+        it('should return the correct step completion status status when all executors are applying', (done) => {
+            const ctx = {
+                'list': [
```

Comment:
Please remove the quotes from object params in these tests to keep consistency throughout the file

Reply:
Ha good spot, I actually pasted it from some other file so will search for that and remove the quotes from there as well.

Reply:
I found the same all over the place... removing them all.